psi4/driver/procrouting/proc.py

```diff
@@ -2138,6 +2138,7 @@ def run_occ(name, **kwargs):
         ref_wfn.semicanonicalize()
 
     occ_wfn = core.occ(ref_wfn)
+    raise Exception
```

Comment:
Is this meant to be here?

Reply:
Anything in the commit marked "Debug" is my attempt to debug a very annoying Windows-specific bug and will not be part of the final PR.